opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -279,30 +285,31 @@ impl UserEventsExporter {
                 eb.add_str("_typeName", "Log", FieldFormat::Default, 0);
                 cs_b_count += 1;
 
-                if log_record.body().is_some() {
-                    eb.add_str(
-                        "body",
-                        // TODO: Use proper type instead of String always.
-                        match log_record.body().as_ref().unwrap() {
-                            AnyValue::Int(value) => value.to_string(),
-                            AnyValue::String(value) => value.to_string(),
-                            AnyValue::Boolean(value) => value.to_string(),
-                            AnyValue::Double(value) => value.to_string(),
-                            AnyValue::Bytes(value) => String::from_utf8_lossy(value).to_string(),
-                            // TODO: Handle complex types using serde_json
-                            AnyValue::ListAny(_value) => "".to_string(),
-                            AnyValue::Map(_value) => "".to_string(),
-                            &_ => "".to_string(),
-                        },
-                        FieldFormat::Default,
-                        0,
-                    );
-                    cs_b_count += 1;
-                }
-                if level != Level::Invalid {
-                    eb.add_value("severityNumber", otel_severity as i16, FieldFormat::SignedInt, 0);
```

Comment:
seems we were already exporting `otel_severity`. What exactly is the change?

Reply:
removed the if() check that was unnecessary.